src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,18 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override bool CursorGrabbed
```

Comment:
The Linux backend only support full screen "windows". I'm not sure if we support multi monitor with it even, something to look into, but I think for now it's ok to just hardcode this to true.